app/scripts/modules/core/src/notification/notificationList.directive.html

```diff
@@ -62,8 +62,8 @@
     <div class="col-md-9 text-right">
         <button class="btn btn-primary ng-scope" ng-if="notificationListCtrl.isNotificationsDirty"
                 ng-click="notificationListCtrl.saveNotifications()">
-            <span class="far fa-check-circle"></span> Save Changes
-            </span></button>
+            <span class="far fa-check-circle">Save Changes</span>
```

Comment:
Maybe double check the space here isnt needed for visual whitespace?

Reply:
I'll just put it back in